internal/stack/boot.go

```diff
@@ -128,12 +137,18 @@ func dockerComposeUp(daemonMode bool, stackVersion string) error {
 		"up",
 	}
 
-	if daemonMode {
+	if options.DaemonMode {
 		args = append(args, "-d")
 	}
 
+	if len(options.Services) > 0 {
+		for _, aService := range options.Services {
+			args = append(args, aService, fmt.Sprintf("%s_is_ready", aService))
```

Comment:
Nit: you can just append `fmt.Sprintf("%s_is_ready", aService)`. Because of the dependency, `aService` will automatically be spun up.

Reply:
I can't, I checked it :) 

With single `_is_ready` service (e.g. elasticsearch_is_ready), the docker-compose will exit immediately when the "true"-application stops, so you can't achieve an effect of tailing live logs. It will behave exactly like in daemon mode.

Reply:
Ah, I see. Good to know — thanks!